test/integration/lerna-publish.test.js

```diff
@@ -66,6 +69,36 @@ describe("lerna publish", () => {
     expect(commitMessage).toMatchSnapshot("commit: updates fixed versions");
   });
 
+  test.concurrent("uses detault suffix with canary flag", async () => {
+    const cwd = await initFixture("PublishCommand/normal");
+    const args = [
+      "publish",
+      "--canary",
+      "--skip-npm",
+      "--yes",
+    ];
+
+    const { stdout, stderr } = await execa(LERNA_BIN, args, { cwd });
+    const hash = await lastCommitId(cwd);
+    expect(stdout.replace(new RegExp(hash, 'g'), 'hash')).toMatchSnapshot("stdout: canary default version");
+    expect(stderr).toMatchSnapshot("stderr: canary beta version");
```

Comment:
This snapshot label should be `stderr: canary default version`, looks like copy+paste oversight.

Reply:
Fixed